packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -73,9 +74,11 @@ const ProfilePage = () => {
   const { lockApp, unlockApp } = useOverlayBlocker();
   const { notifyStatus } = useNotifyAT();
   const { currentTheme, themes: allApplicationThemes, onChangeTheme } = useThemeToggle();
+  const { get } = useFetchClient();
+
   useFocusWhenNavigate();
 
-  const { status, data } = useQuery('user', () => fetchUser(), {
+  const { isLoading: isLoadingUser, data } = useQuery('user', fetchUser, {
```

Comment:
Almost :) Now could you please inline the contents of `fetchUser`?

Reply:
done

Reply:
I don't see it?

Reply:
ok

Reply:
done